__tests__/app.test.js

```diff
@@ -190,29 +190,29 @@ describe("/api/users", () => {
         });
     });
 });
-// describe("/api/articles", () => {
-//     describe('GET, 200: returns all articles', () => {
-//         test('returns an array of 3 objects with the expected keys', () => {
-//             return request(app)
-//                 .get('/api/articles')
-//                 .expect(200)
-//                 .then(({ body }) => {
-//                     expect(body.articles).toBeInstanceOf(Array);
-//                     expect(body.articles.length).toBe(12);
-//                     body.articles.forEach((article) => {
-//                         expect(article).toBeInstanceOf(Object);
-//                         expect(article).toMatchObject({
-//                             article_id: expect.any(Number),
-//                             title: expect.any(String),
-//                             topic: expect.any(String),
-//                             author: expect.any(String),
-//                             body: expect.any(String),
-//                             created_at: expect.any(String),
-//                             votes: expect.any(Number),
-//                             comments: expect.any(Number)
-//                         });
-//                     });
-//                 });
-//         });
-//     });
-// });
\ No newline at end of file
+describe("/api/articles", () => {
+    describe('GET, 200: returns all articles', () => {
+        test('returns an array of 12 objects with the expected keys', () => {
+            return request(app)
+                .get('/api/articles')
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.articles).toBeInstanceOf(Array);
+                    expect(body.articles.length).toBe(12);
+                    body.articles.forEach((article) => {
+                        expect(article).toBeInstanceOf(Object);
+                        expect(article).toMatchObject({
+                            article_id: expect.any(Number),
+                            title: expect.any(String),
+                            topic: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            comment_count: expect.any(String)
+                        });
+                    });
+                });
+        });
+    });
```

Comment:
Cant see any tests/code for the queries bit of the ticket 

**Queries**
The end point should also accept the following query:
- topic, which filters the articles by the topic value specified in the query. If the query is omitted the endpoint should respond with all articles.